src/network.rs

```diff
@@ -1,73 +1,73 @@
-use std::io;
-use std::net::{IpAddr, Ipv4Addr, SocketAddr, TcpListener, TcpStream};
+use std::net::{IpAddr, Ipv4Addr, SocketAddr, TcpStream};
 use std::time::Duration;
 
 use anyhow::Result;
-use gtk4::glib::error;
-use log::error;
+use log::{debug, error};
 use pnet::datalink;
-use pnet::ipnetwork::IpNetwork;
-use pnet::packet::ipv4::Ipv4;
 
 use crate::client;
 use crate::client::ElgatoClient;
 
 pub fn get_active_devices() -> Vec<ElgatoClient> {
-    let mut active_devices = vec![];
-    let network = get_interface();
-    let devices = get_devices(network.unwrap());
-    for device in devices.iter() {
-        println!("Device: {:?}", device);
-        active_devices.append(&mut vec![client::ElgatoClient::new(device.to_string().as_str())]);
-        break;
-    }
-    active_devices
+  let mut active_devices = vec![];
+  let network = get_interface();
+  let devices = get_devices(network.unwrap());
+
+  #[allow(clippy::never_loop)]
+  for device in devices.iter() {
+    debug!("Device: {:?}", device);
+    active_devices.append(&mut vec![client::ElgatoClient::new(
+      device.to_string().as_str(),
+    )]);
+    break;
```

Comment:
As there is this break in the loop clippy complains that this never loops, but I think this is just temporary, right? Otherwise it would be wiser to remove

Reply:
This is temporary. Since I was doing a full scan .0/24 it was taking too long to finish.